### PR TITLE
change reviewed/unreviewed feature - Service level

### PR DESCRIPTION
**Pull request by NoyRobinson**

### CODE REVIEW

File: src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt [hunk 1]
```diff
@@ -78,4 +78,13 @@ class FeedbackService(private val feedbackDao: FeedbackDao) {
 
         } ?: throw NotFoundException("Feedback not found")
     }
+
+    fun changeToReviewedOrUnreviewed(feedbackId: Int, companyId: Int, employeeId: Int, reviewed: Boolean) {
+        val feedbackInfo = feedbackDao.getFeedbackById(feedbackId, companyId)
```
Comment (vasofi) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt]: think if you really need this.
This means that you are going to the db twice - once for fetch, once for update. is it necessary?

Comment (NoyRobinson) [reply src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt]: It's unnecessary. I removed it

File: src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt [hunk 2]
```diff
@@ -78,4 +78,9 @@ class FeedbackService(private val feedbackDao: FeedbackDao) {
 
         } ?: throw NotFoundException("Feedback not found")
     }
+
+    fun changeToReviewedOrUnreviewed(feedbackId: Int, companyId: Int, employeeId: Int, reviewed: Boolean) {
```
Comment (vasofi) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt]: we don't like 'or' and 'and' in the function names. Usually it's a code smell that the function does more than one thing. Also - if you had more than 2 options, you wouldn't put them all in the function name.
You can name - changeReviewedStatus or something like that

Comment (vasofi) [reply src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt]: Same in the dao

**Review by vasofi — APPROVED** [review]
rename the func

**Pull request merged**